src/features/moderation/useIsAdmin.tsx

```diff
@@ -1,6 +1,12 @@
+import { Person } from "lemmy-js-client";
 import { useAppSelector } from "../../store";
-import { isAdminSelector } from "../auth/siteSlice";
 
-export default function useIsAdmin() {
-  return useAppSelector(isAdminSelector);
+export default function useIsAdmin(person?: Person) {
```

Comment:
What is the purpose of this change?

Reply:
I needed a way to determine admin status for arbitrary users so I can properly highlight them in the modlog (see ModLogItem.tsx:171). 

This is basically the previous version of this function (from before it was replaced with a static selector in ead60d3), augmented by an optional `person` parameter.

Since it's not used anywhere else right now, I can make this a private function in `ModLogItem` instead and restore this helper to its last state if you prefer.

Reply:
I see. How about something like `useAppSelector(!person ? isAdminSelector : state => state.site.response?.admins?.some(
      (admin) => admin.person.id === person?.id
    ))` instead?

Reply:
Sounds good, can do.